packages/core/src/routes/application.ts

```diff
@@ -69,14 +85,14 @@ export default function applicationRoutes<T extends AuthedRouter>(
       );
 
       if (paginationDisabled) {
-        ctx.body = await findApplications(search, excludeApplicationIds);
+        ctx.body = await findApplications(search, excludeApplicationIds, types);
```

Comment:
double check we have an issue to remove the `excludeApplicationIds` parameter (use excludeRoleId instead)